app/helpers/budgets_helper.rb

```diff
@@ -22,6 +22,10 @@ def budget_phases_select_options
     Budget::Phase::PHASE_KINDS.map { |ph| [ t("budgets.phase.#{ph}"), ph ] }
   end
 
+  def budget_voting_styles_select_options
+    Budget::Vote::KINDS.map { |vk| [ t("budgets.voting_style.#{vk}"), vk] }
```

Comment:
We try to use full names instead of acronyms, makes life happier 😌
¿How about using a variable name `kind` or `style`?

Reply:
Yah, Ruby generally doesn't use acronyms. I think `style` makes more sense. Will update.